blosc/blosc2.c

```diff
@@ -43,7 +43,11 @@
   #include <ippdc.h>
 #endif
 #if defined(HAVE_ZLIB_NG)
+#ifdef ZLIB_COMPAT
   #include "zlib.h"
+#else
+  #include "zlib-ng.h"
+#endif
```

Comment:
By selecting "zlib-ng.h" you are introducing yet another format to Blosc2, which is not what we'd like to do.  We have chosen zlib-ng as a modern implementation of zlib, but that was always meant to be run in ZLIB_COMPAT mode.  What we want is to use the external zlib-ng, but in ZLIB_COMPAT mode.  Is that possible?